code/_macros.dm

```diff
@@ -19,6 +19,8 @@
 
 #define iscarbon(A) istype(A, /mob/living/carbon)
 
+#define isnecromorph(A) istype(A, /mob/living/carbon/human/necromorph)
```

Comment:
we have datum.is_necromorph(), use that

If you absolutely need a macro version, make it a wrapper to call that.
It's much more complex, and covers a variety of odd cases like divider heads taking over a body